mysql-test/suite/rocksdb/t/lock_wait_timeout_stats.test

```diff
@@ -0,0 +1,26 @@
+--source include/have_rocksdb.inc
+
+#
+## MyRocks-specific tests for locking
+#
+#--source include/have_debug.inc
+
+--enable_connect_log
+create table t (a int not null primary key) engine=rocksdb;
+
+begin;
+insert into t values (0);
+
+--source include/count_sessions.inc
+--connect (con1,localhost,root,,)
+--connection con1
+set @@rocksdb_lock_wait_timeout=1;
+begin;
+--error ER_LOCK_WAIT_TIMEOUT
+insert into t values(0);
+select INNODB_ROW_LOCK_WAIT_TIMEOUTS from information_schema.table_statistics where table_engine="ROCKSDB" and table_name="t";
+select INNODB_ROW_LOCK_WAIT_TIMEOUTS from information_schema.table_statistics where table_engine="ROCKSDB" and table_name="t";
```

Comment:
Now that we are incrementing properly, let's add a few more queries that cause timeouts from different connections and verify the stats go up properly, rather just query twice in a row where the stats don't change like this.